pylutron/__init__.py

```diff
@@ -162,10 +178,13 @@ def _main_loop(self):
           line = t.read_until(b"\n", timeout=3)
         else:
           raise EOFError('Telnet object already torn down')
-      except (EOFError, TimeoutError, socket.timeout, AttributeError):
+      except _EXPECTED_NETWORK_EXCEPTIONS:
+        _LOGGER.exception("Uncaught exception")
```

Comment:
also one thought here is it might be nicer to differentiate the log event from the full failure (this one catches and continues, vs main which aborts)